chromadb/test/test_api.py

```diff
@@ -92,6 +92,37 @@ def fastapi_server():
     print("Including integration tests")
     test_apis.append(fastapi_integration_api)
 
+@pytest.mark.parametrize("api_fixture", [local_persist_api])
```

Comment:
@HammadB do you mind taking a look at this when you get a chance? I can get a "scriptified" test to pass/fail with this code change, but not the test. The test always passes.

Reply:
I believe api = request.getfixturevalue(api_fixture.__name__) caches the local value and returns it to you, so you are likely not reloading the client.

Reply:
https://docs.pytest.org/en/6.2.x/fixture.html#fixtures-can-be-requested-more-than-once-per-test-return-values-are-cached

Reply:
Thanks for the tip! Will take it from here :)